### DISCUSSION TITLE
Website Doesn't Work Properly on Mobile Browser

### DISCUSSION THREAD

**Discussion by TheMercenary333** [community/community#42601] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

On iPhone 6 IOS 12.5.6, the existing Chrome 71.0.3578.89 and Safari; on General Mobile e-Tab 4 Android 4.2.2 Mozilla 68.11.0, the Github site opens, but the Assets section under the release page shared for the applications does not load, only the loading circle rotates. Also, the same problem occurs in the branches section. 
*Actually, it was working before. But now it is not working. I think there has been a change on the site. That's why it broke. 

![photo_5855193292910738825_x](https://user-images.githubusercontent.com/74915480/209471252-19692f58-1fc8-4c73-8ce8-8216f9872854.jpg)

![photo_5855193292910738832_x](https://user-images.githubusercontent.com/74915480/209471951-4b269522-6ceb-4c8b-9332-9a7aba0b0719.jpg)

Also, when I click on the three lines in a row that I marked in red, no section opens. It is not functional. 
![photo_5855193292910738831_x](https://user-images.githubusercontent.com/74915480/209471958-c9d8e374-ca05-4882-9722-8d514b57d766.jpg)

**Comment by stevepopovich** [comment]
Hi @TheMercenary333 This category is specifically for feedback for the native mobile app. I will move this to the appropriate place!

**Comment by dmytro-shchurov** [comment]
It seems that the website doesn’t work at all in a mobile browser (any of them!). And nobody cares for years. A typical situation when a public product becomes an enterprise one.

And GitHub Mobile for some reasons needs iOS 15 (which I don’t want install on a device that was not designed for that version).
